larpandoracontent/LArThreeDReco/LArHitCreation/ThreeDHitCreationAlgorithm.h

```diff
@@ -328,7 +423,7 @@ inline ThreeDHitCreationAlgorithm::ProtoHit::ProtoHit(const pandora::CaloHit *co
 
 inline const pandora::CaloHit *ThreeDHitCreationAlgorithm::ProtoHit::GetParentCaloHit2D() const
 {
-    return m_pParentCaloHit2D;
+    return m_pParentCaloHit2D; // TODO: I've now basically changed this by allowing nullptr. Should check its usage.
 }
```

Comment:
Will check if the default empty constructor is still needed...I can't remember which part needed this, and if I can remove it again which would be preferable.